server/core/src/main/java/dev/slimevr/tracking/processor/HumanPoseManager.kt

```diff
@@ -641,6 +641,10 @@ class HumanPoseManager(val server: VRServer?) {
 	val userHeightFromConfig: Float
 		get() = skeletonConfigManager.userHeightFromOffsets
 
+	@get:ThreadSafe
+	val realUserHeight: Float
+		get() = skeletonConfigManager.userHeightFromOffsets / 0.936f
```

Comment:
You can reference [BodyProportionError.kt](https://github.com/SlimeVR/SlimeVR-Server/blob/main/server/core/src/main/java/dev/slimevr/autobone/errors/BodyProportionError.kt#L34) for this value